install/README.md

```diff
@@ -219,6 +220,47 @@ A list of _datalogger_ sampling configurations for a given _station_ and recordi
 | _Start_ | Stream start time|
 | _Stop_ | Stream stop time|
 
+#### _POLARITIES_ ####
+
+Site specific times when the recorded values may have a reversed, or otherwise, polarity.
+This is often difficult to track with the known instrument responses and installation details as it can
```

Comment:
Why couldn't this go in the sensors.csv? Alongside information like azimuth, dip?

Reply:
hello  @staylorofford  .. :) 
i would say because it doesn't always has to deal with the sensor but also the datalogger or the cables (or unknown forgotten  reason for a reversal...)

Reply:
I'm not 100% sure where to go here.

Either we keep it as is (i.e. the polarity is in the streams, as the reversed column) and have this table more of formal-ish documentation as to why.

Or we switch completely over, and the polarity table is used (which then introduces the problems related to possibly inconsistent or conflicting data).

Reply:
It makes sense to have this separate, as @salichon says. Can we have this table as the source of truth and keep the other files free of polarity information?